src/selectors/entities/channels.test.js

```diff
@@ -9,6 +9,884 @@ import {sortChannelsByDisplayName, getDirectChannelName} from 'utils/channel_uti
 import * as Selectors from 'selectors/entities/channels';
 import {General, Preferences} from '../../constants';
 
+const sortUsernames = (a, b) => a.localeCompare(b, General.DEFAULT_LOCALE, {numeric: true});
+
+describe('Selectors.Channels.getChannelsInCurrentTeam', () => {
+    const team1 = TestHelper.fakeTeamWithId();
+    const team2 = TestHelper.fakeTeamWithId();
+
+    it('should return channels in current team', () => {
+        const user = TestHelper.fakeUserWithId();
+
+        const profiles = {};
+        profiles[user.id] = user;
+
+        const channel1 = TestHelper.fakeChannelWithId(team1.id);
+        channel1.display_name = 'Channel Name';
```

Comment:
What's the reason for setting the display name on all these?

Reply:
I set them because the ordering use in theory the display_name. Nevertheless, I removed them and the result is the same, so they are not needed.